projects/map/src/lib/helpers/ol-layer.helper.ts

```diff
@@ -160,4 +179,18 @@ export class OlLayerHelper {
       });
     }
   }
+
+  public static getWmsServiceParams(layer: WMSLayerModel): WmsServiceParamsModel {
+    const params: WmsServiceParamsModel = {
+      LAYERS: layer.layers,
+      VERSION: '1.1.1',
+      QUERY_LAYERS: layer.queryLayers,
+      TRANSPARENT: 'TRUE',
+    };
+    if (layer.filter) {
+      params.CQL_FILTER = layer.filter;
```

Comment:
at some point we probably want to make this a function call that takes `layer.filter` and the `ServerType` or `ResolvedServerType` as an argument and returns the vendor specific filter parameter as noted in https://b3partners.atlassian.net/browse/HTM-463?focusedCommentId=39331